src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -133,24 +134,28 @@ def _get_scripted_model(self, pt_module, example_inputs=None):
         import inspect
 
         def prepare_example_inputs(inputs, input_signature):
+            is_torch_2 = parse(torch.__version__) >= parse("2.0.0")
             if inputs is not None:
                 if isinstance(inputs, dict):
+                    ordered_inputs = []
                     if input_signature is not None:
-                        ordered_inputs = []
                         used_sign = []
                         for key in input_signature:
                             if key not in inputs:
                                 continue
                             ordered_inputs.append(inputs[key])
                             used_sign.append(key)
-                        inputs = ordered_inputs
                         input_signature = used_sign
                     else:
-                        inputs = list(inputs.values())
-                        input_signature = input_signature[:len(inputs)]
+                        ordered_inputs = list(inputs.values())
+                    if is_torch_2:
+                        return {"example_kwarg_inputs": inputs}, input_signature
```

Comment:
example_kwarg_inputs ([dict](https://docs.python.org/3/library/stdtypes.html#dict), optional) – This parameter is a pack of keyword arguments of example inputs that will be passed to the function while tracing. Default: None. Either this argument or example_inputs should be specified. The dict will be unpacking by the arguments name of the traced function. If the keys of the dict don’t not match with the traced function’s arguments name, a runtime exception will be raised.

it is used in trorch.jit.trace
https://pytorch.org/docs/stable/generated/torch.jit.trace.html

Reply:
this dict represents input_parameters in code where we call trace and passed as kwargs to torch.jit.trace. I used such format for minimization difference of call between regular inputs case and kwargs

Reply:
can we use `example_inputs` instead?

Reply:
what do you mean? Just different name of variable or use another parameter for tracing.
If second, no it may lead to incorrect results in case when forward has optional arguments ad provided not all inputs with skipping in middle.
I provided test for this case (when we have input x, y=None, z=None and provided dict with x and z only. using example_nputs will lead that model got for tracing x and y as it accept list of unnamed inputs